src/config.rs

```diff
@@ -1459,6 +1459,14 @@ pub struct Delta<T: Default> {
     pub y: T,
 }
 
+#[derive(Clone, Copy, Debug, Default, Deserialize)]
+pub struct Padding {
+    pub top: u8,
+    pub right: u8,
+    pub bottom: u8,
+    pub left: u8,
```

Comment:
It might make sense to add `x` and `y` to this struct as a deprecation strategy, so no current configuration files are broken.